qsimcirq/qsim_simulator.py

```diff
@@ -99,7 +99,7 @@ def _needs_trajectories(circuit: cirq.Circuit) -> bool:
                 op, {param: 1 for param in cirq.parameter_names(op)}
             )
         )
-        if not (cirq.has_unitary(test_op) or cirq.is_measurement(test_op)):
+        if not (cirq.is_measurement(test_op) or cirq.has_unitary(test_op)):
```

Comment:
So Cirq tries to apply the measurement operation to determine if it is unitary or not, right? Perhaps `cirq.has_unitary` should be updated for measurements.

Reply:
This is a good suggestion - opened https://github.com/quantumlib/Cirq/issues/4959 to track.